lib/rspec/core/lookup_set.rb

```diff
@@ -0,0 +1,32 @@
+module RSpec
+  module Core
+    # @private
+    class LookupSet
+      include Enumerable
+
+      def initialize(array=[])
+        @values = {}
+        merge(array)
+      end
+
+      def <<(key)
+        @values[key] = true
+      end
```

Comment:
With a normal set (and every other built-in that defines a `<<` method that I can think of), this method returns the set itself:

```
irb(main):002:0> s = Set.new
=> #<Set: {}>
irb(main):003:0> s << 1
=> #<Set: {1}>
```

Yours returns `true`, I believe.  I think you should add `self` on the next line to mirror the normal behavior.